tests/jni_api.rs

```diff
@@ -600,6 +592,27 @@ pub fn throw_new_fail() {
     assert_pending_java_exception(&env);
 }
 
+#[test]
+pub fn throw_defaults() {
+    let env = attach_current_thread();
+
+    assert_throws(&env, "Default exception thrown");
```

Comment:
Why don't we instead of throwing just use `JThrowable.lookup(env, "foo")` and then assert on the exception object (type + message)?

Reply:
Well, I wanted also to cover the `.throw()` itself (we have tests for `.throw_new()`, but nothing for `.throw()`), but probably it's not the best idea.